dnn/generated_algorithmperformance.go

```diff
@@ -11,13 +11,13 @@ type AlgorithmPerformance struct {
 	internal C.cudnnAlgorithmPerformance_t
 
 	algoDesc *AlgorithmDescriptor
-	status   Status
+	status   *cudnnStatus
 	time     float32
 	memory   uintptr
 }
 
 // NewAlgorithmPerformance creates a new AlgorithmPerformance.
-func NewAlgorithmPerformance(algoDesc *AlgorithmDescriptor, status Status, time float32, memory uintptr) (retVal *AlgorithmPerformance, err error) {
+func NewAlgorithmPerformance(algoDesc *AlgorithmDescriptor, status *cudnnStatus, time float32, memory uintptr) (retVal *AlgorithmPerformance, err error) {
```

Comment:
This should not change. This should be `Status`. Remember you don't want people who are using the library to suddenly have to drop in to type `C.XXX`

Reply:
Actually, we should create a type alias `type Status = cudnnStatus`